src/main/java/com/deloitte/elrr/datasync/dto/MessageVO.java

```diff
@@ -1,5 +1,7 @@
 package com.deloitte.elrr.datasync.dto;
```

Comment:
so...just throwing this out there. If Message is just a POJO, and all it contains is statement, why not just use statement and get rid of this class?